applications/crossbar/priv/couchdb/schemas/tasks.records.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json-schema.org/draft-03/schema#",
+    "_id": "tasks.records",
+    "additionalProperties": false,
```

Comment:
Do these objects get stored in the DB in a way that a client can access the object later? If so, consider removing this restriction as clients like to store custom data along side Kazoo data.